app/gui/view/src/project.rs

```diff
@@ -795,7 +799,7 @@ impl application::View for View {
             (Press, "", "cmd alt p", "toggle_component_browser_private_entries_visibility"),
             (Press, "", "cmd s", "save_project_snapshot"),
             (Press, "", "cmd shift r", "restore_project_snapshot"),
-            (Press, "", "cmd z", "undo"),
+            // (Press, "", "cmd z", "undo"),
```

Comment:
Commented shortcut. Quite important, I'd say.

Reply:
Oh, good catch. It was left after debugging.